### PR TITLE
remove lint from Makefile. It could give false alarms.

### PR DESCRIPTION
**Pull request by iamqizhao**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,13 +38,7 @@ proto:
...
 	done
 
-lint: testdeps
-	go get -v github.com/golang/lint/golint
-	for file in $$(git ls-files '*.go' | grep -v '\.pb\.go$$' | grep -v '_string\.go$$'); do \
-		golint $$file; \
-	done
-
-pretest: lint
+pretest:
```
Comment (dsymonds) [inline Makefile]: it doesn't seem like you need the pretest target either.

**Comment by dsymonds** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (iamqizhao) [inline Makefile]: On Fri, Jun 12, 2015 at 6:08 PM, David Symonds <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/grpc/grpc-go/pull/225#discussion_r32366036>:
>
> > @@ -39,13 +38,7 @@ proto:
> >  		protoc -I $$(dirname $$file) --go_out=plugins=grpc:$$(dirname $$file) $$file; \
> >  	done
> >
> > -lint: testdeps
> > -	go get -v github.com/golang/lint/golint
> > -	for file in $$(git ls-files '*.go' | grep -v '\.pb\.go$$' | grep -v '_string\.go$$'); do \
> > -		golint $$file; \
> > -	done
> > -
> > -pretest: lint
> > +pretest:
>
> it doesn't seem like you need the pretest target either.
>
It is in a pending commit. I am trying to figure out why travis is broken
on this.

Comment (iamqizhao) [inline Makefile]: On Fri, Jun 12, 2015 at 6:10 PM, Qi Zhao <toqizhao@gmail.com> wrote:

>
>
> On Fri, Jun 12, 2015 at 6:08 PM, David Symonds <notifications@github.com>
> wrote:
>
>> In Makefile
>> <https://github.com/grpc/grpc-go/pull/225#discussion_r32366036>:
>>
>> > @@ -39,13 +38,7 @@ proto:
>> >  		protoc -I $$(dirname $$file) --go_out=plugins=grpc:$$(dirname $$file) $$file; \
>> >  	done
>> >
>> > -lint: testdeps
>> > -	go get -v github.com/golang/lint/golint
>> > -	for file in $$(git ls-files '*.go' | grep -v '\.pb\.go$$' | grep -v '_string\.go$$'); do \
>> > -		golint $$file; \
>> > -	done
>> > -
>> > -pretest: lint
>> > +pretest:
>>
>> it doesn't seem like you need the pretest target either.
>>
> It is in a pending commit. I am trying to figure out why travis is broken
> on this.
>
https://travis-ci.org/grpc/grpc-go/builds/66626132

Comment (iamqizhao) [inline Makefile]: ok, I got it.

**Pull request merged**